spec/requests/users_spec.rb

```diff
@@ -1,4 +1,34 @@
 require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe 'Users', type: %w[request] do
+  before :each do
+    @user = User.new(
+      name: 'Killy',
+      photo: 'http://great.com/org.pjg',
+      bio: 'Full-stack developer',
+      posts_counter: 0
+    )
+    @user.save
+  end
+
+  describe 'GET /index' do
+    it 'renders index template' do
+      get '/users/'
+      expect(response.status).to eq(200)
+      expect(response).to render_template('index')
```

Comment:
- [ ] Great job creating the tests for users requests however you should include tests that ensure the response body includes the correct placeholder text, you did this in the next test but kindly make sure to include it also in this test. e.g `expect(response.body).to include(.....)`